integration/simulation/simulation_docker_test.go

```diff
@@ -25,19 +23,25 @@ import (
 
 // TODO - Use individual Docker containers for the Obscuro nodes and Ethereum nodes.
 
-var (
+const (
 	enclaveDockerImg  = "obscuro_enclave"
 	nodeIDFlag        = "--nodeID"
 	addressFlag       = "--address"
 	enclaveAddress    = ":11000"
 	enclaveDockerPort = "11000/tcp"
+	dockerTestEnv     = "DOCKER_TEST_ENABLED"
 )
 
 // This test creates a network of L2 nodes, then injects transactions, and finally checks the resulting output blockchain
 // The L2 nodes communicate with each other via sockets, and with their enclave servers via RPC.
 // All nodes live in the same process, the enclaves run in individual Docker containers, and the Ethereum nodes are mocked out.
 // $> docker rm $(docker stop $(docker ps -a -q --filter ancestor=obscuro_enclave --format="{{.ID}}") will stop and remove all images
 func TestDockerNodesMonteCarloSimulation(t *testing.T) {
+	fooAddr := os.Getenv(dockerTestEnv)
```

Comment:
"fooAddr"?

Reply:
changing it to something more meaningful :)